src/installing-workflow/index.md

```diff
@@ -0,0 +1,95 @@
+you are just getting started with Kubernetes and Deis Workflow, follow our [quickstart guide](../quickstart/index.md)
+with help booting Kubernetes and installing Workflow.
+
+
+## Prerequisites
+
+You need to [install Helm Classic and Deis Workflow CLI before continuing](../quickstart/install-cli-tools.md).
+
+## Check Your Setup
+
+Check that the `helmc` command is available and the version is 0.7 or newer.
+
+```
+$ helmc --version
+helmc version 0.7.0+20a7ed7
+```
+
+Ensure the `kubectl` client is installed and can connect to your Kubernetes cluster. `helmc` uses `kubectl` to interact
+with your Kubernetes cluster.
+
+You can test that helmc and kubectl are working properly by running:
+
+```
+$ helmc target
+Kubernetes master is running at https://52.9.206.49
+Elasticsearch is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/elasticsearch-logging
+Heapster is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/heapster
+Kibana is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kibana-logging
+KubeDNS is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kube-dns
+kubernetes-dashboard is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
+Grafana is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana
+InfluxDB is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb
+```
+
+If you see a list of targets like the one above, `helmc` can communicate with the Kubernetes master. Also make sure that
+kubectl is pointing at the cluster you expect.
+
+## Add the Deis Chart Repository
+
+The [Deis Chart Repository](https://github.com/deis/charts) contains everything you need to install Deis Workflow onto
+your Kubernetes cluster, with a single `helmc install` command.
+
+Add this repository to Helm Classic:
+
+```
+$ helmc repo add deis https://github.com/deis/charts
+```
+
+## Install Deis Workflow
+
+Now that you have Helm Classic installed and have added the Deis Chart Repository, install Workflow by running:
+
+```
+$ helmc fetch deis/workflow-beta4             # fetches the chart into a
+                                              # local workspace
+$ helmc generate -x manifests workflow-beta4  # generates various secrets
+$ helmc install workflow-beta4                # injects resources into
+                                              # your cluster
+```
+
+Helm Classic will install a variety of Kubernetes resources in the `deis` namespace.
+You'll need to wait for the pods that it launched to be ready. Monitor their status
+by running:
+
+```
+$ kubectl get pods --namespace=deis
+```
+
+If you would like `kubectl` to automatically update as the pod states change, run (type Ctrl-C to stop the watch):
+```
+$ kubectl get pods --namespace=deis -w
```

Comment:
Minor point: if you always put the `--namespace=` flag immediately after `kubectl` it makes it easier to alias:
```console
$ kubectl --namespace=deis get pods --watch
$ alias kd='kubectl --namespace=deis'
$ kd get po -w
```